src/Provider/LazySchemaDiffProvider.php

```diff
@@ -34,6 +45,16 @@ public function createToSchema(Schema $fromSchema): Schema
             return LazySchema::createLazyProxy(static fn () => $originalSchemaManipulator->createToSchema($fromSchema));
         }
 
+        if (PHP_VERSION_ID >= 80400) {
+            $reflector = new ReflectionClass(Schema::class);
+
+            if ($reflector->isUninitializedLazyObject($fromSchema)) {
+                return $reflector->newLazyProxy(
+                    static fn () => $originalSchemaManipulator->createToSchema($fromSchema),
```

Comment:
PHP complains that this object is lazy, and I do not understand why.

Reply:
OK actually, it was, although `isUninitializedObject()` says the oppositte, because it's a clone: https://github.com/doctrine/migrations/blob/dfc28aea72cc28da34dafdd4c9e3e24abd703947/src/Provider/DBALSchemaDiffProvider.php#L35-L38

So now I just don't understand why `isUninitializedObject()` said the opposite, but anyway, there is no need to check, because `initializeLazyObject()` can be used on non-lazy objects (it's a no-op).